app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,18 @@
+.desk-card{
+  margin: 0.5rem;
+}
+
+.desk-card-col{
+  max-width: 15em;
+  min-width: 15em;
+}
+
+.desk-row{
+  padding-left: 5em;
+  padding-right: 5em
+}
+
+body > .alert, body > .alert > .close
+{
+  padding: 0.3rem 1.25rem 0.3rem 1.25rem;
+}
```

Comment:
Could we make the CSS styling more consistent. Here's a reference [style guide](https://github.com/airbnb/css#formatting)

I see a couple of things:
1. space between selector and `{`
2. `{` on same line as selector
3. comma separated selectors on their own lines
4. semi colon on line 12

Reply:
Thoughts?

Reply:
Thanks Dom, I agree. Will do.